holiday-calendar-eur/src/test/java/com/github/davejoyce/calendar/impl/HolidayCalendarServiceUSTest.java

```diff
@@ -0,0 +1,31 @@
+package com.github.davejoyce.calendar.impl;
+
+import com.github.davejoyce.calendar.HolidayCalendar;
+import com.github.davejoyce.calendar.HolidayCalendarFactory;
+import com.github.davejoyce.calendar.HolidayCalendarService;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class HolidayCalendarServiceUSTest {
+
+    static final String CODE = "US";
```

Comment:
Ah, so code is country code?  I feel like regional variations have the potential to be problematic, although you may well have thought through all that already :).  Maybe if code does mean country then that needs to be made explicit and validated against the appropriate ISO 3166 standard?

Reply:
@battleblow actually, code _does not_ necessarily equate to country code. This library must support non-national holiday calendars, too -- e.g. [SIFMA](https://www.sifma.org/resources/general/holiday-schedule/), [Federal Reserve](https://www.federalreserve.gov/aboutthefed/k8.htm), other central banks like the ECB.